parm/FV3.input.yml

```diff
@@ -374,6 +391,7 @@ FV3_GFS_v16:
     <<: *gfs_gfdl_cloud_mp
     mp_time: 150.0
     reiflag: 2
+    rthresh: 1.0e-06
```

Comment:
These changes will change the answer for all configurations. We need to make sure that that is welcome, and that all configurations using this package will pass the test.

Reply:
This is the recommendation of the physics group at EMC.

Reply:
@yangfanglin, could you explain why this value is better than the previous one (1.0e-05)? @christinaholtNOAA is worried about this change.

Reply:
@chan-hoo  Correction:   In your table you shared with me the two values are actually the same,  rthresh = 1e-05 in SRWapp and rthresh = 10.e-6 in global workflow.    The default value in model code is 10.e-6

Reply:
Removed.